docs/examples/gestures/.gitignore

```diff
@@ -70,3 +70,8 @@
 !**/ios/**/default.pbxuser
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+lib/generated_plugin_registrant.dart
+macos/
+test/
+web/
```

Comment:
Following #332 this change won't be necessary.

Reply:
Indeed, but I am not going to remove this yet, as we are not sure which PR will land first